vacationtime-solution/README.md

```diff
@@ -0,0 +1,70 @@
+# Manage Employee Vacation Time Requests
+
+_Quickly approve/disapprove employees' vacation time requests, and 
+create calendar events or send notification emails accordingly.._
+
+Last updated: July, 2019
+
+A Google Form is a great tool for employees to quickly submit vacation
+time requests in a uniform manner. However, it can be difficult for employers
+to deal with everybody's requests at once, and send the correct e mails/create 
+the correct calendar events accordingly.
+
+This solution uses [Google Apps Script][apps-script] to generate a Google Sheet
+and apropriate columns based on employees' vacation time requests. When an employee
+submits a request through a Google Form, the employer solely needs to load the Form
+responses into a Google Sheet, run the Apps Script code, and mark approval/disapproval
+column values as necessary. The script will then notify employees of their disapproved
+requests through automated emails, or create a shared calendar event between
+manager and employee for those with approved requests.
+Now, managers can easily see on their calendars when certain employees will be on
+vacation, and have dealt with all other requests appropriately and with the simple
+click of a button.
+
+[apps-script]: https://developers.google.com/apps-script
+
+## Technology highlights
+
+- The script sends automated e mails to employees with 'NOT APPROVED' status
+  using [MailApp][mail-app].
+- The script creates shared calendar events between manager and employee using
+  [CalendarApp][cal-app].
+- The script sets certain columns to contain drop-down menu cells using 
+  [Data Validation rules][data-val].
+
+[mail-app]: https://developers.google.com/apps-script/reference/mail/mail-app
+[cal-app]: https://developers.google.com/apps-script/reference/calendar/calendar-app
+[data-val]: https://developers.google.com/apps-script/reference/spreadsheet/data-validation-builder
+
+
+## Try it out!
+
+First, set up your Google Form:
+1. Create a copy of the sample [Employee Time Off Request Form][request-form].
+1. Once you've received responses, on your google form hit **Edit > View Summary***. Click the three summary dots in the top right corner, and click on **Select Response Destination** from the drop-down.
+
+Second, set up your Google Sheet:
+1. Create a new Google Sheet and name it whatever you'd like. The Form responses will now appear in this sheet.
+1. On your Google Sheet, go to **Tools > Script Editor**
+1. Copy + Paste the given Code.gs code into the Script Editor, and hit Save. Now refresh your spreadsheet.
+
+Lastly, run the AppsScript functions:
+1. A dialog box will appear and tell you that the script requires authorization. Read the authorization notice and continue.
+1. You will see a new **TimeOff** menu. Click it and select **Manager Approval**. In the last column you will now be able to approve/disapprove each employee’s request.
```

Comment:
See comment in code about renaming to 'Set up' to make it clear it's a one-time thing.